modules/fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/fs2/io/Sinks.scala

```diff
@@ -37,12 +37,34 @@ object Sinks {
 
   /**
    * Set the delay threshold to use for batching. After this amount of time has elapsed (counting
-   * from the first element added), the elements will be wrapped up in a batch and sent. This
-   * value should not be set too high, usually on the order of milliseconds. Otherwise, calls
-   * might appear to never complete.
+   * from the first element added), the elements will be wrapped up in a batch and sent.
+   *
+   * If the source MaxQueueSize is small, then the queue could block between batches if
+   * DelayThreshold is too large.  If the source MaxQueueSize is sufficiently large, then the
+   * DelayThreshold should not ever cause blocking.
    */
   val DelayThreshold: FiniteDuration = 200.milliseconds
 
+  /**
+   * A batch of messages will be emitted to PubSub when the batch reaches 1000 messages.
+   * We use 1000 because it is the maximum batch size allowed by PubSub.
+   * This overrides the permutive library default of `5`
+   */
+  val PubsubMaxBatchSize = 1000L
```

Comment:
Does the library handle the fact that with these number of messages we can reach >10MB ?